File: ACVPWorkflow/Models/WorkflowItemLite.cs
using System;

namespace ACVPWorkflow.Models
{
    public class WorkflowItemLite
    {
        public long WorkflowItemId { get; set; }
        public long RequestId { get; set; }
        public APIAction APIAction { get; set; }
        public string SubmissionId { get; set; }
        public string Submitter { get; set; }
        public DateTime Submitted { get; set; }
        public WorkflowStatus Status { get; set; }
    }
}

File: Web.Admin/ClientApp/src/app/models/workflow/WorkflowItemLite.ts
import { APIAction } from './APIAction.enum';
import { WorkflowStatus } from './WorkflowStatus.enum';

export class WorkflowItemLite {
  workflowItemId: number;
  requestId: number;
  apiAction: APIAction;
  submissionId: string;
  submitter: string;
  submitted: Date;
  status: WorkflowStatus;

  // This is used for the multi-select on the Workflows page only
  multiSelected: boolean;
  multiSelectSubmissionStatus: string;
  multiApproveErrorReviewMessage: string;
}
